tests/Solana/TransactionTests.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "PublicKey.h"
+#include "Solana/Address.h"
+#include "Solana/Transaction.h"
+
+#include "BinaryCoding.h"
+
+#include <gtest/gtest.h>
+
+using namespace TW;
+using namespace TW::Solana;
+
+TEST(SolanaTransaction, TransferMessageData) {
+    auto from = Address("6eoo7i1khGhVm8tLBMAdq4ax2FxkKP4G7mCcfHyr3STN");
+    auto to = Address("56B334QvCDMSirsmtEJGfanZm8GqeQarrSjdAb2MbeNM");
+    Solana::Hash recentBlockhash("11111111111111111111111111111111");
+    auto transaction = Transaction(from, to, 42, recentBlockhash);
+    Data expected_message = {
```

Comment:
can we also use hex strings in this file?

Reply:
Oh yes, definitely. Sorry for missing it!

Reply:
Updated and rebased